doc/api/tracing.md

```diff
@@ -23,28 +20,91 @@ The available categories are:
     measurements.
 * `v8`
 
-By default the `node`, `node.async_hooks`, and `v8` categories are enabled.
-
 ```txt
-node --trace-events-enabled --trace-event-categories v8,node,node.async_hooks server.js
+node --trace-event-categories v8,node,node.async_hooks server.js
+```
+
+```js
+const trace_events = require('trace_events');
+trace_events.enableTracingCategories('v8', 'node,node.async_hooks');
 ```
 
 Running Node.js with tracing enabled will produce log files that can be opened
-in the [`chrome://tracing`](https://www.chromium.org/developers/how-tos/trace-event-profiling-tool)
-tab of Chrome.
+in the [`chrome://tracing`][] tab of Chrome.
 
 The logging file is by default called `node_trace.${rotation}.log`, where
 `${rotation}` is an incrementing log-rotation id. The filepath pattern can
 be specified with `--trace-event-file-pattern` that accepts a template
 string that supports `${rotation}` and `${pid}`. For example:
 
 ```txt
-node --trace-events-enabled --trace-event-file-pattern '${pid}-${rotation}.log' server.js
+node --trace-event-file-pattern '${pid}-${rotation}.log' server.js
 ```
 
 Starting with Node.js 10.0.0, the tracing system uses the same time source
 as the one used by `process.hrtime()`
 however the trace-event timestamps are expressed in microseconds,
 unlike `process.hrtime()` which returns nanoseconds.
 
+Previous versions of Node.js required use of the `--trace-events-enabled`
+command line flag to enable trace event capture. This flag is no longer
+required to enable trace events. If it is used, however, trace events are
+enabled using the default categories: `v8`, `node`, and `node.async_hooks`.
+
+## The `trace_events` module
+
+The `trace_events` module may be used to turn trace event reporting on and off.
+
+It is accessible using `require('trace_events')`.
+
+### trace_events.disableTracingCategories(...categories)
+
+* `...categories` [string] One or more category names.
```

Comment:
`[string]` -> `{string}` (otherwise this will not be linkified by the type parser).